src/Controller/ClickController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Click;
+use App\Entity\Stats;
+use Doctrine\Persistence\ManagerRegistry;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ClickController extends AbstractController
+{
+    private ManagerRegistry $doctrine;
+
+    public function __construct(ManagerRegistry $doctrine)
+    {
+        $this->doctrine = $doctrine;
+    }
+
+    #[Route('/click', name: 'click')]
+    public function indexAction()
+    {
+        $linkId = $_GET['linkId'];
+        $link = $this->doctrine->getManager()->find('App\Entity\Link', $linkId);
```

Comment:
```suggestion
        $link = $this->doctrine->getManager()->find(Link::class, $linkId);
```
I presume this was left on purpose. If so - please ignore my comment.